### PR TITLE
Update with transactions and queries

### PR DESCRIPTION
**Pull request by shasherazi**
In this PR, I updated the database in a controlled manner using transactions. I also ran some aggregate functions to answer some meaningful questions. Here are the screenshots of the updates and queries:
<details>
<summary> Screenshots </summary>

Screenshot-2023-03-28_13:59:11
Screenshot-2023-03-28_14:06:12
Screenshot-2023-03-28_14:08:55
Screenshot-2023-03-28_14:13:13
Screenshot-2023-03-28_16:04:13
Screenshot-2023-03-28_16:04:21
Screenshot-2023-03-28_16:04:36
Screenshot-2023-03-28_16:05:04
Screenshot-2023-03-28_16:05:19
Screenshot-2023-03-28_16:05:39
</details>

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Sasherazi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that has incorrect gitflow/ GitHub flow. Kindly note that the Pull Request for this activity should flow **from feature into development**.
- Kindly also ensure that the _Future Features_ section is present in your ReadMe including 1-3 potential features to be added later. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by shasherazi** [comment]
My bad, this was a simple project, so I thought I'd just use the default. I fixed it now.

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,80 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+----------------------------------------------
+
+-- update the animals table by setting the species column to unspecified
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
```
Comment (uwadonat) [inline queries.sql:18]: - [x] I can see there is no changes in your `Schema.sql` file. As requested in project requirement, let's modify your `schema.sql` file by adding a column `species` of type `string` to your `animals` table.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @shasherazi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### highlights
-  Correct workflow was used ✔️
-  Descriptive pull request message ✔️
-  Good commit messages ✔️
almost

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (shasherazi) [reply queries.sql:18]: Can't believe I missed that :sweat_smile:. All fixed now.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔🍾**
Hello @shasherazi 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Good implementation of suggested changes.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**